EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRaw.cc

```diff
@@ -145,176 +146,178 @@ namespace sistrip {
 	  
 	  //need to construct full object to copy full header
 	  std::unique_ptr<const sistrip::FEDBuffer> fedbuffer;
-	  //std::unique_ptr<const sistrip::FEDBufferBase> bufferBase;
-	  try {
-	    fedbuffer.reset(new sistrip::FEDBuffer(rawfedData.data(),rawfedData.size(),true));
-	    //bufferBase.reset(new sistrip::FEDBufferBase(rawfedData.data(),rawfedData.size()));
-	  } catch (const cms::Exception& e) {
-	    edm::LogWarning("DigiToRaw") << "[sistrip::DigiToRaw::createFedBuffers_]"
-		      << " Invalid raw data, cannot get buffer..."
-		      << std::endl;
-	  }
-	  
-	  if ( edm::isDebugEnabled() ) {
-	    edm::LogWarning("DigiToRaw")
-	      << "[sistrip::DigiToRaw::createFedBuffers_]"
-	      << " Original header type: " << fedbuffer->headerType();
-	  }
-
-	  bufferGenerator_.setHeaderType(FEDHeaderType::HEADER_TYPE_FULL_DEBUG);
-	  //fedbuffer->headerType());
-	  bufferGenerator_.daqHeader() = fedbuffer->daqHeader();
-	  bufferGenerator_.daqTrailer() = fedbuffer->daqTrailer();
-	  
-	  bufferGenerator_.trackerSpecialHeader() = fedbuffer->trackerSpecialHeader();
-	  bufferGenerator_.setReadoutMode(mode_);
-
-	  if ( edm::isDebugEnabled() ) {
-	    std::ostringstream debugStream;
-	    if(ifed==fed_ids.begin()){  bufferGenerator_.trackerSpecialHeader().print(std::cout); std::cout << std::endl;}
-	    bufferGenerator_.trackerSpecialHeader().print(debugStream);
-	    edm::LogWarning("DigiToRaw")
-	      << "[sistrip::DigiToRaw::createFedBuffers_]"
-	      << " Tracker special header apveAddress: " << static_cast<int>(bufferGenerator_.trackerSpecialHeader().apveAddress())
-	      << " - event type = " << bufferGenerator_.getDAQEventType()
-	      << " - buffer format = " << bufferGenerator_.getBufferFormat() << "\n"
-	      << " - SpecialHeader bufferformat " << bufferGenerator_.trackerSpecialHeader().bufferFormat()
-	      << " - headertype " << bufferGenerator_.trackerSpecialHeader().headerType()
-	      << " - readoutmode " << bufferGenerator_.trackerSpecialHeader().readoutMode()
-	      << " - apvaddrregister " << std::hex << static_cast<int>(bufferGenerator_.trackerSpecialHeader().apvAddressErrorRegister())
-	      << " - feenabledregister " << std::hex << static_cast<int>(bufferGenerator_.trackerSpecialHeader().feEnableRegister())
-	      << " - feoverflowregister " << std::hex << static_cast<int>(bufferGenerator_.trackerSpecialHeader().feOverflowRegister())
-	      << " - statusregister " << bufferGenerator_.trackerSpecialHeader().fedStatusRegister() << "\n"
-	      << " SpecialHeader: " << debugStream.str();
-	  }
+          if ( rawfedData.size() == 0 ) {
```

Comment:
the code appears to be indented inconsistently. Please correct.

https://github.com/cms-sw/cmssw/blob/19e622e49166d7b3749e1befcdb45fc7cf1dac43/EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRaw.cc#L147-L150